tests/tests.cpp

```diff
@@ -98,17 +98,54 @@ TEST_CASE("test if bad data is inputted", "[weight=1][part=1]")
 //     REQUIRE(true);
 // }
 
-// // Advanced test cases
-// TEST_CASE("testTraverse1", "[weight=1][part=2]")
-// {
-//     // testing traversing the tree for the shortest path (dataset 1)
-//     REQUIRE(true);
-// }
-// TEST_CASE("testTraverse2", "[weight=1][part=2]")
-// {
-//     // testing traversing the tree for the shortest path (dataset 2)
-//     REQUIRE(true);
-// }
+// Advanced test cases
+TEST_CASE("testTraverse1", "[weight=1][part=2]")
```

Comment:
can you also be descriptive on the name of the test? like what does it do?

Reply:
fixed